src/barril/units/posc.py

```diff
@@ -204,6 +204,11 @@ def FillUnitDatabaseWithPosc(db=None, fill_categories=True, override_categories=
     db.AddUnitBase(
         "standard volume per volume", "std cubic metres, 15 deg C/cubic metre", "scm(15C)/m3"
     )
+    db.AddUnitBase(
+        "standard volume per standard volume",
+        "std cubic metres, 15 deg C/std cubic metre",
```

Comment:
I think here it should be

```suggestion
        "std cubic metres, 15 deg C/std cubic metre, 15 deg C",
```